topics/statistics/tutorials/dome/tutorial.md

```diff
@@ -13,11 +13,530 @@ key_points:
+
+
+The recommendations mentioned above were initially developed by the [ELIXIR Machine Learning Focus Group](https://elixir-europe.org/focus-groups/machine-learning) in response to a published Comment advocating for the establishment of standards in ML for biology. 
+This focus group, comprising over 50 experts in the field of ML, held meetings to collaboratively develop and refine the recommendations through broad consensus.
+
+In the following chapters the publication from [@MobiDB] is going to be used as an example.
+
+
+
+[//]: # (From Docs: Chapter 01)
+
+
+ML models analyze experimental biological data by identifying patterns, which can then be used to generate biological insights from similar, previously unseen data. 
+The ability of a model to maintain its performance on new data is referred to as its generalization power. 
+Achieving strong generalization is a key challenge in developing ML models; without it, trained models cannot be effectively reused. 
+Properly preprocessing data and using it in an informed way are essential steps to ensure good generalization.
+
+
+
+
+> **Note: Further Reading**  
+> State-of-the-art ML models are often capable of memorizing all variations within the training data. As a result, when evaluated on data from the training set, they may give the false impression of excelling at the given task. However, their performance tends to diminish when tested on independent data (called the test or validation set), revealing a lower generalization power. To address this issue, the original dataset should be randomly split into non-overlapping parts. The simplest method involves creating separate training and test sets (with a possible third validation set). Alternatively, more robust techniques like cross-validation or bootstrapping, which repeatedly create different training/testing splits from the available data, are often preferred.
+	
+	Handling overlap between training and test data can be especially challenging in biology. For instance, in predicting entire gene or protein sequences, ensuring data independence might require reducing homologs in the dataset. In modeling enhancer–promoter contacts, a different criterion may be needed, such as ensuring that no endpoint is shared between training and test sets. Similarly, modeling protein domains may require splitting multidomain sequences into their individual domains before applying homology reduction. Each biological field has its own methods for managing overlapping data, making it crucial to consult prior literature when developing an approach.
```

Comment:
Having these indents at the start of a sentence turns it into a code block, which I don't think you wanted.